dls-controls/malcolmjs
# Feature/attribute plotting #265

## Description
Added plot and table view for attribute data
## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/gui/PANDA:INENC1/val
- view the graph widget
- click on "table" tab at the bottom of the screen and view the archive table
## Agile board tracking

connect to #265 
connect to #264
